build.gradle

```diff
@@ -101,8 +101,8 @@ task clean(type: Delete) {
 
 ext {
     minSdkVersion = 21
-    targetSdkVersion = 31
-    compileSdkVersion = 31
+    targetSdkVersion = 33
```

Comment:
might be worth also bumping app compat to 1.5.x in this same PR?

Reply:
Yes, can do that as well.